server/src/test/java/org/elasticsearch/common/collect/ImmutableOpenMapTests.java

```diff
@@ -120,6 +131,58 @@ public void testEmptyKeySetWorks() {
         assertThat(ImmutableOpenMap.of().keySet().size(), equalTo(0));
     }
 
+    public void testEmptyValuesIsCollection() {
+        assertTrue(ImmutableOpenMap.of().values().isEmpty());
+    }
+
+    public void testValuesIsCollection() {
+        assertTrue(countryPopulations.values().containsAll(List.of(37_846_611, 46_754_778, 60_461_826, 65_273_511, 83_783_942)));
+    }
+
+    public void testValuesToArray() {
+        Integer[] populations = countryPopulations.values().toArray(Integer[]::new);
+
+        assertThat(populations.length, equalTo(5));
+        Arrays.sort(populations);
+        assertThat(populations, equalTo(new Integer[]{37_846_611, 46_754_778, 60_461_826, 65_273_511, 83_783_942}));
```

Comment:
I think we should use the ham rest collection matcher here because the order is not guaranteed